catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,547 @@
+local id = 'USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/TRENDS_YEAR';
+local subdir = 'USGS';
+local version = '5';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO(AG): Remove skip_indexing when the preview image is ready.
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'RCMAP Rangeland Component Timeseries V5 Trends Year (1985-2021)' + version,
+  version: version,
+  'gee:type': ee_const.gee_type.imagecollection,
+
+  description: |||

```

Comment:
@simonff what do I need to change here? I've tried to redo this so many times, it has to be a simple fix. Thanks